general-concepts/virtuals/text.xml

```diff
@@ -42,5 +42,113 @@ from the Gentoo repository.
 </note>
 </body>
 
+<section>
+<title>KEYWORDS in virtual packages</title>
+
+<body>
+<p>
+Since virtual packages do not install any files, they do not follow the regular
+arch testing procedure. Instead, the developer can immediately set
+their <c>KEYWORDS</c> to the union of <c>KEYWORDS</c> of their providers.
+In particular, if a new virtual is created for a stable package, the virtual is
+committed straight to stable.
+</p>
+
+<p>
+For example, if you have two packages: <c>dev-libs/liblinux</c> with
+<c>KEYWORDS="amd64 ~x86"</c> and <c>dev-libs/libbsd</c> with
+<c>KEYWORDS="~amd64-fbsd ~x86-fbsd"</c>, the resulting virtual will
+have:
+</p>
+
+<codesample lang="ebuild">
+KEYWORDS="amd64 ~x86 ~amd64-fbsd ~x86-fbsd"
+
+RDEPEND="|| ( dev-libs/liblinux dev-libs/libbsd )"
+</codesample>
+</body>
+
+</section>
+
+<section>
+<title>Virtuals and subslots</title>
+
+<body>
+<warning>
+This section is only applicable if virtual providers include versions that
+are ABI-compatible with one another (and use matching SONAMEs) and/or
+the incompatible providers are being obsoleted.
+</warning>
+
+<p>
+Alike regular packages, virtuals can define subslots that can be used
+to trigger rebuilds of their reverse dependencies. For this to work, a new
+version of the virtual is created for each subslot of the providers, and it
```

Comment:
I think this sentence is a bit ambiguous. How about we rewrite the part following `, and` like this: `, and in each version specific subslots are listed in dependencies`. The plurality of multiple "versions" doesn't play well with `it`. Even something like: `For this to work, a new version of the virtual is created for each subslot of the providers, where each version contains dependencies for a specific subslot` clarifies it I think.